packages/core-lib/src/space-connector/helper.ts

```diff
@@ -61,12 +61,21 @@ class ApiQueryHelper extends QueryHelper {
         return this;
     }
 
+    // This method is temporarily added for v2 api.
+    setMultiSortV2(sortList: Sort[]): ApiQueryHelper {
+        if (!this._data.sort) {
+            this._data.sort = {};
+        }
+        this._data.sort = sortList;
+        return this;
+    }
+
```

Comment:
suggestion: What about to add a `setApiVersion` method and consolidate the functionalities of `setMultiSortV2` and `setMultiSort` within a single `setMultiSort` method with conditional handling for v2 API, as needed? 
This approach would avoid the need to add separate methods for each new API version, making the code more efficient.